config/samples/numaplane.numaproj.io_v1alpha1_pipelinerollout.yaml

```diff
@@ -1,9 +1,10 @@
 apiVersion: numaplane.numaproj.io/v1alpha1
 kind: PipelineRollout
 metadata:
-  name: pipelinerollout-example
-  namespace: numaplane-system
+  name: my-pipeline
+  namespace: example-namespace
 spec:
+  interStepBufferServiceName: my-isbsvc
```

Comment:
This needs to be added to the pipelinerollout_types.go file in the PipelineRolloutSpec struct.

Reply:
oh, sorry, this is a typo: I meant to put it underneath `pipeline` below

Reply:
good catch